main.go

```diff
@@ -48,6 +48,7 @@ var (
 	metricsAddr   = flag.String("metrics_addr", ":8095", "configure http listener for reporting metrics")
 	enableProfile = flag.Bool("enable-pprof", false, "enable pprof profiling")
 	debugAddr     = flag.String("debug_addr", "localhost:6060", "port for pprof profiling")
+	writeSecrets  = flag.Bool("write_secrets", true, "whether to write the secrets directly (true) or returns secrets in grpc response (false, requires driver v0.20.1)")
```

Comment:
Nit: 'directly' -> 'directly to the filesystem'

Reply:
done